python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -1019,30 +1019,34 @@ def create_reexecuted_run_from_failure(
         parent_run: PipelineRun,
         repo_location: "RepositoryLocation",
         external_pipeline: "ExternalPipeline",
-        tags: Optional[Dict[str, Any]] = None,
+        extra_tags: Optional[Dict[str, Any]] = None,
```

Comment:
I think technically a breaking change to rename this, but I think this is okay.  I'd definitely call this out in the release notes though.

Reply:
ah, nevermind... this is brand new, right?

Reply:
It went out in 0.14.9 and not in release notes or anything, can't imagine someone is using it already